docs/clawback/clawback-of-issued-currency.md

```diff
@@ -6,18 +6,18 @@ labels:
   - Tokens
 status: not_enabled
 ---
+
 # Clawback
-<embed src="/snippets/_clawback-disclaimer.md" />
-For regulatory purposes, some issuers must have the ability to recover issued tokens after they are distributed to accounts. For example, if an issuer were to discover that tokens were sent to an account sanctioned for illegal activity, the issuer could recover, or *claw back*, the funds.
+
+&#x20;For regulatory purposes, some issuers must have the ability to recover issued tokens after they are distributed to accounts. For example, if an issuer were to discover that tokens were sent to an account sanctioned for illegal activity, the issuer could recover, or \*claw back\*, the funds.
```

Comment:
Isn't &#x20; just a space? Also, escaping the asterisks on `*claw back*` is definitely a mistake; that's _supposed_ to be italics there.